changes/2240.removal.rst

```diff
@@ -0,0 +1 @@
+If you have a Pyside6 app deployed to macOS, you must add ``min_os_version = "12.0"`` to your macOS project configuration. As of PySide 6.8.0, PySide6 macOS wheels are tagged with a minimum supported macOS version of 12.0. Previously, Briefcase would install the macOS 12 wheel, but the Briefcase app would declare itself as supporting macOS 11. This would cause errors if the app was run on macOS 11. Briefcase will no longer install macOS wheels that are incompatible with the minimum OS version declared by the app (11.0 by default). The additional ``min_os_version`` configuration option is now required to allow Briefcase to resolve the installation of PySide6 wheels.
```

Comment:
```suggestion
If you have a PySide6 app deployed to macOS, you must add ``min_os_version = "12.0"`` to your macOS project configuration. As of PySide 6.8.0, PySide6 macOS wheels are tagged with a minimum supported macOS version of 12.0. Previously, Briefcase would install the macOS 12 wheel, but the Briefcase app would declare itself as supporting macOS 11. This would cause errors if the app was run on macOS 11. Briefcase will no longer install macOS wheels that are incompatible with the minimum OS version declared by the app (11.0 by default). The additional ``min_os_version`` configuration option is now required to allow Briefcase to resolve the installation of PySide6 wheels.
```